scripts/db_migrator.py

```diff
@@ -670,6 +670,24 @@ def migrate_config_db_flex_counter_delay_status(self):
                 flex_counter['FLEX_COUNTER_DELAY_STATUS'] = 'true'
                 self.configDB.mod_entry('FLEX_COUNTER_TABLE', obj, flex_counter)
 
+    def migrate_sflow_table(self):
+        """
+        Handle sflow table migration. Migrations handled:
+        1. 'sample_direction' attr in SFLOW object was newly introduced.
```

Comment:
>1. [](http://example.com/codeflow?start=8&length=2)

Where are the numbers following "1." ?

Reply:
> > 1.
> 
> Where are the numbers following "1." ?

Removed numbering and rephrased the text.